assets/js/pages/Users/UserForm.jsx

```diff
@@ -242,6 +245,26 @@ function UserForm({
               }}
             />
           </div>
+          <Label
+            className="col-start-1 col-span-1"
+            htmlFor="totp"
+            aria-label="totp"
+          >
+            TOTP
+          </Label>
+          <div className="col-start-2 col-span-3">
+            <Select
+              optionsName="totp"
+              options={['Enabled', 'Disabled']}
```

Comment:
Let's put the `Enabled` option disabled.
Like: `options={[{ value: 'Enabled", disabled: true }, 'Disabled']}`.
This way, the user cannot literally choose the `Enabled` button, and it is only selected if it comes from the backend